tests/data/kern/voice_dublifications.krn

```diff
@@ -1,29 +1,29 @@
-**kern		**kern	**kern	**kern
-*staff2		*staff2	*staff1	*staff1
-*clefF4		*clefF4	*clefG2	*clefG2
-*k[f#c#]		*k[f#c#]	*k[f#c#]	*k[f#c#]
-*M4/4		*M4/4	*M4/4	*M4/4
-*b:		*b:	*b:	*b:
-*MM58		*MM58	*MM58	*MM58
-=1-		=1-	=1-	=1-
-[1F#		16A#L	8r	2dd
-.		16F#	.	.
-.		16G#	8f#	.
-.		16A#J	.	.
-.		16BL	8dL	.
-.		16c#	.	.
-.		16d	8BJ	.
-.		[16BJ	.	.
-.		16B]L	8gL	[2cc#
-.		16B	.	.
-.		16A#	8f#J	.
-.		16BJ	.	.
-.		16c#L	8bL	.
-.		16d	.	.
-.		16e	8a#J	.
-.		16c#J	.	.
-=2		=2	=2	=2
-*^		*	*	*
+**kern  **kern	**kern	**kern
```

Comment:
comment abou the file name: I think the correct english term is "duplication"?

Reply:
Addressed with commit [093758d](https://github.com/CPJKU/partitura/pull/344/commits/093758d9e3c4f904e50bbf34615ea46e747214ac)